src/utils/onLink.js

```diff
@@ -72,9 +72,14 @@ const onLink = (config, handler) => {
     // 2. rel="external" attribute
     if (element.hasAttribute('download') || element.getAttribute('rel') === 'external') { return false; }
 
+    const originalHref = element.getAttribute('href');
+    const sanitizedHref = originalHref.toLowerCase();
```

Comment:
For later checks so we are case insensitive.

Reply:
Nice one! One tiny nitpick: I'd call the variable `lowerCasedHref` instead of `sanitizedHref`, because you're not really doing any sanitizing of the href. Very minor, though!